run_tests.py

```diff
@@ -1,58 +0,0 @@
-"""Wrapper for running tests from the commandline."""
-
-from __future__ import annotations
-
-import sys
-from pathlib import Path
-from typing import NoReturn
-
-import click
-
-from nuke_test_runner.configuration import find_configuration, load_runners
-from nuke_test_runner.runner import Runner
-
-
-def run_tests(interpreter: str | Path, tests: str | Path) -> NoReturn:
-    """Run the tests.
-
-    Args:
-        interpreter: name of the configured interpreter or the path to the nuke executable.
-        tests: path to the test file/folder.
-    """
-    runner = None
-
-    if isinstance(interpreter, str):
-        search_start = Path(tests.split("::")[0])
-        config = find_configuration(search_start)
-        if config:
-            runners = load_runners(config)
-            runner = runners.get(interpreter)
-
-    runner = runner or Runner(interpreter)
-    sys.exit(runner.execute_tests(tests))
-
-
-@click.command()
-@click.argument("interpreter", type=str)
-@click.argument("tests", type=str)
-def main(interpreter: str, tests: str) -> NoReturn:
-    """Run the test files with nuke.
-
-    INTERPRETER is the filepath to the nuke executable.
-    If you provided a "runner.json" configuration,
-    you can also reference the runner by its configured name.
-
-    TESTS is the folder of file of the tests you want to execute.
```

Comment:
All the args have their associated help documentation included. I've also now added a more detailed description:

```text
Nuke Test Runner CLI Interface.

  This bootstraps Nuke within the test runner to be able to run pytest like
  usual, with all Nuke dependencies.

  `nuke-executable` is the filepath to the nuke executable. If you provided a
  "runner.json" configuration, you can also reference the runner by its
  configured name.

  `test-dir` is the folder of file of the tests you want to execute. Use the
  pytest folder/file.py::class::method notation to run single tests. For
  further options consult the pytest documentation.

  If you don't specify the test-dir it will use the current directory like
  pytest does.

Options:
  -n, --nuke_executable PATH     Path to the executable of Nuke.
  -t, --test_dir PATH            Directory to test. This defaults to the
                                 current directory and will search for tests
                                 in that folder recursively like pytest does.
  -c, --config PATH              Specify a json to read as config to use for
                                 the tests.
  -r, --runner_name TEXT         Only run the runners in the config specified
                                 with this name
  -i, --run_interactive BOOLEAN  Launch a Nuke interpreter to run the tests
                                 in. This defaults to True.
  -p, --pytest_arg TEXT          Specify an arg to forward to pytest. You can
                                 add as many of these as you want.
  --help                         Show this message and exit.
  ```